pkg/messaging/mqtt/setup_test.go

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) Mainflux
+// SPDX-License-Identifier: Apache-2.0
+
+package mqtt_test
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"os/signal"
+	"syscall"
+	"testing"
+	"time"
+
+	"github.com/mainflux/mainflux/logger"
+	"github.com/mainflux/mainflux/pkg/messaging"
+	"github.com/mainflux/mainflux/pkg/messaging/mqtt"
+	dockertest "github.com/ory/dockertest/v3"
+)
+
+const (
+	protocol = "mqtt"
+	id       = "mqtt-publisher"
+	timeout  = time.Second * 5
+	qos      = 1
+)
+
+var (
+	publisher  messaging.Publisher
+	subscriber messaging.Subscriber
+)
+
+func TestMain(m *testing.M) {
+	pool, err := dockertest.NewPool("")
+	if err != nil {
+		log.Fatalf("Could not connect to docker: %s", err)
+	}
```

Comment:
Could you replace this by:
```
	require.Nil(t, err, fmt.Sprintf("Could not connect to docker: %s", err))
```

Reply:
No, because I use `m *testing.M` as we normally do, to an argument to `TestMain` function.